ginga/canvas/types/basic.py

```diff
@@ -158,7 +159,7 @@ def contains_arr(self, x_arr, y_arr):
         # See: http://alienryderflex.com/polygon/
         xa, ya = x_arr, y_arr
 
-        result = numpy.empty(ya.shape, dtype=numpy.bool)
+        result = numpy.empty((ya.size, xa.size), dtype=numpy.bool)
```

Comment:
Without this "fix", I get the following error:
```
2015-09-01 10:16:24,820 | E | Callback.py:89 (make_callback) | Traceback:
  File ".../ginga-2.4.20150827203255-py2.7.egg/ginga/misc/Callback.py", line 72, in make_callback
    res = method(*cb_args, **cb_kwdargs)

  File "~/.ginga/plugins/SurfBgRatio.py", line 390, in update
    self.draw_cb(canvas, tag)

  File "~/.ginga/plugins/SurfBgRatio.py", line 469, in draw_cb
    return self.redo()

  File "~/.ginga/plugins/SurfBgRatio.py", line 307, in redo
    sci_masked = image.cutout_shape(sig_obj)

  File ".../ginga-2.4.20150827203255-py2.7.egg/ginga/BaseImage.py", line 330, in cutout_shape
    view, mask = self.get_shape_view(shape_obj)

  File ".../ginga-2.4.20150827203255-py2.7.egg/ginga/BaseImage.py", line 318, in get_shape_view
    contains = shape_obj.contains_arr(xi, yi)

  File ".../ginga-2.4.20150827203255-py2.7.egg/ginga/canvas/types/basic.py", line 181, in contains_arr
    result[tf == True] ^= cross[tf == True]
```

This is the info I get when I added a print function in `canvas/types/basic.py`:
```python
>>> print(xa.shape, ya.shape, tf.shape, result.shape, cross.shape)
(1, 136), (131, 1), (131, 136), (131, 1), (131, 136)
```